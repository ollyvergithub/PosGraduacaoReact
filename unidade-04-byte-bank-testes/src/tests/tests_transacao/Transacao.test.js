import React from "react";
import {render} from "@testing-library/react";
import Transacao from "../../transacoes/Transacao";

describe('Componente de Transação do extrato', () =>{

    it('Deve retornar sempre o mesmo snapshot', function () {

        const {container} = render(
            <Transacao
                data='07/08/2021'
                tipo = 'saque'
                valor = '50.00'
            />
        )
        expect(container.firstChild).toMatchSnapshot()
    });

})