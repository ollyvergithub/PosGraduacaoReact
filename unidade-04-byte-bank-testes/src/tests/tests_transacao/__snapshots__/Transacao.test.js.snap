// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de Transação do extrato Deve retornar sempre o mesmo snapshot 1`] = `
<div
  class="transacao-container"
>
  <p>
    07/08/2021
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    50.00
  </p>
</div>
`;
